crates/apub/src/activities/mod.rs

```diff
@@ -190,45 +186,41 @@ where
   Url::parse(&id)
 }
 
+pub(crate) trait GetActorType {
+  fn actor_type(&self) -> ActorType;
+}
```

Comment:
Im not sure why this trait is needed, can you explain it?

Reply:
The reason is that the rows in sent_activity now need to be associated to a specific actor (for signing). so I added an actor_id to the table. But we also need to be able to deserialize that actor. So we also need to know what kind of actor it is. That's why I added `actor_type` to the db, so when loading activities it is known what kind of actor it is and where to get it from.